Haitian Creole

```diff
@@ -0,0 +1 @@
+Mwen leve yon maten, mwen jwenn tout panno chanm nan tounen dous makos.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.